### ISSUE TITLE
Improve Font Color for Better Readability

### ISSUE THREAD

**Issue by megh-bari** [anuragverma108/WildGuard#1661]
Description:
The current font color on the website is set to grey, which lacks sufficient contrast against the background, making text hard to read for many users. This is impacting overall readability and accessibility, particularly for users with visual impairments.

![image](https://github.com/user-attachments/assets/02736c6c-7be6-4198-b1dd-65897869e1a0)

kindly assigned this issue to hacktoberfest 2024!
